hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/contracts/ParsingConstants.java

```diff
@@ -40,12 +40,15 @@ private ParsingConstants() {
     public static final String UINT256 = "(uint256)";
 
     // struct types
+    // Defined by struct Expiry { unint32 second; address autoRenewAccount; uint32 autoRenewPeriod; }
```

Comment:
Good comment, we can also consider linking the `IHederaTokenService` [struct](https://github.com/hashgraph/hedera-smart-contracts/blob/main/contracts/hts-precompile/IHederaTokenService.sol#L69) when appropriate.